src/resources/styles/mui-variables-mapping.scss

```
@@ -57,6 +57,10 @@
 	--oriso-app-action: #a5000a;
 	--oriso-lottie-accent-color: var(--oriso-primary-fixed-dim, #ffb4aa);
 	--oriso-lottie-secondary-color: var(--m3-secondary-container, #646d78);
+	// Confirmation animations sit on white and carry meaning, so they take the
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)